doc/content/getting-started/migrating-from-networks/device-json.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Create Device.json"
+title: "Create devices.json"
```

Comment:
Probably change to a title that describes the process instead, like "Export End Devices"